main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 			}
 			wl = _wl
 		} else if config.DiceConfig.Persistence.WALEngine == "aof" {
-			_wl, err := wal.NewAOFWAL(config.DiceConfig.Persistence.WALDir)
+			_wl, err := wal.NewAOFWAL(config.DiceConfig.Persistence.WALDir, false, 4096 * 16000, 10, "size", 5 * time.Second, true, "default")
```

Comment:
These are a lot of parameters to initiate the `WALAOF` object. We can define a `WALConfig` object and have a default config set